native/README.md

```diff
@@ -0,0 +1,5 @@
+The native driver (the one producing the native AST) should be located here.
```

Comment:
I guess this file can be removed or changed to reflect that this is a working native driver and not a template.

Reply:
Yes, this was added when I did the `bblfsh-sdk update`, I'll remove it (don't think this is one of the files checked to exists).